src/lib/core/backtrace.cc

```diff
@@ -74,49 +74,75 @@ backtrace_proc_cache_clear(void)
 	proc_cache = NULL;
 }
 
-const char *
-get_proc_name(unw_cursor_t *unw_cur, unw_word_t *offset, bool skip_cache)
+static int
+backtrace_proc_cache_find(unw_word_t ip, const char **name, unw_word_t *offset)
```

Comment:
I would like to discuss this point.

From your point of view, could we provide _descriptive_ and _concise_ enough function names (if you consider the current ones are not), so that they would document themself and FWIW (DRY and documentation maintenance) save us from the need to provide such comments?

Reply:
Came to an agreement that descriptive and concise comments are helpful and follows the _double-entry bookkeeping_ principle.